chore: add routes for drag-and-drop code examples

Blackbaud-SteveBrush opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/blackbaud/skyux?pullRequest=157) <br/>All committers of the pull request should sign our Contributor License Agreement in order to get your pull request merged.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: Blackbaud-PatrickOFriel<br/>:x: Travis<hr/>**Travis** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account ([for further information, click here] (https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user)).

Blackbaud-SteveBrush committed:
feat(migrations)!: add support for Angular v13 (#85)

Ran migration to support Nx 13 and Angular 13. Dropped support for Angular 12.

BREAKING CHANGE: Drop support for Angular 12

Blackbaud-SteveBrush committed:
chore(release): 6.0.0-beta.0

Blackbaud-SteveBrush committed:
fix(components/theme): add SCSS variables and mixins to package exports (#92)

Blackbaud-SteveBrush committed:
feat(migrations): add migrate schematic to fix SCSS tilde imports (#94)

Blackbaud-SteveBrush committed:
chore(release): 6.0.0-beta.1

bradenbiz committed:
fix(components/tabs): fix disappearing URL params (#77) (#95)

Blackbaud-SteveBrush committed:
fix(migrations): install `@angular/cdk` if relevant packages installed (#96)

Blackbaud-SteveBrush committed:
chore: update `dev-infra` (#97)

johnhwhite committed:
fix(components/lookup): fix loop caused by highlighting empty search string (#91) (#101)

Blackbaud-SteveBrush committed:
docs: add overview content to dropdown (#98) (#100)

Co-authored-by: Paula Hodgkins <77297281+blackbaud-paulah@users.noreply.github.com>

Blackbaud-SteveBrush committed:
fix(migrations): remove extensions from tilde imports (#102)

Blackbaud-SteveBrush committed:
chore(release): cherry-pick 5.7.2 release notes (#103)

Blackbaud-SteveBrush committed:
chore(release): 6.0.0-beta.2

johnhwhite committed:
fix(components/ag-grid): ag-grid: display validator popover using a delayed hover event (#87) (#110)

Blackbaud-TrevorBurch committed:
feat: update country field and phone field dependencies (#111) (#112)

Blackbaud-SteveBrush committed:
chore: fix npm run test:affected (#109)

Blackbaud-RonFrum committed:
fix(components/lists): add ARIA label to pagination link button (#99)

Adding an ARIA label used for the pagination button to navigate to a specific page in order to
improve accessibility.

Blackbaud-SteveBrush committed:
docs: add modal classes content, fix misspellings (#104) (#106)

Co-authored-by: Paula Hodgkins <77297281+blackbaud-paulah@users.noreply.github.com>

johnhwhite committed:
feat(components/ag-grid): add text cell maxlength and number cell min/max options (#113) (#115)

* feat(components/ag-grid): add text cell maxlength and number cell min/max options

* Add test coverage and code examples

* Code example navigation

Co-authored-by: Mukul Sharma <mukul.sharma@blackbaud.me>

Co-authored-by: Mukul Sharma <mukul.sharma@blackbaud.me>

Blackbaud-SteveBrush committed:
revert(migrations): remove 'fix-scss-tilde-imports' schematic (#105)

This schematic is moved to our internal Angular Builders library since it does not concern our
components, specifically.

Blackbaud-SteveBrush committed:
fix(components/data-manager): assign specific types to properties of `SkyDataViewConfig` (#89)

Assign more specific types for the properties `additionalOptions`, `onClearAllClick`, and
`onSelectAllClick` of `SkyDataViewConfig`. This was done to address the
`@typescript-eslint/ban-types` ESLint rule. Marking as a breaking change, but this will likely not
cause any downstream effects.

BREAKING CHANGE: The properties `additionalOptions`, `onClearAllClick`, and `onSelectAllClick` of
`SkyDataViewConfig` are assigned more specific types.

johnhwhite committed:
feat(components/core): create a resize observer service as a media query service for modals (#70) (#116)

* feat(components/core): create a resize observer service as a media query service for modals

* Provide resize observer as media query service within modals

* Formatting and coverage

* Update test coverage, changes from feedback

* Add examples for resize observer

* Fix code examples app-component test

* Add playground examples

* Fix imports

* Formatting

* Updates based on feedback

* Formatting

* Formatting

* Simplify modal setup. Drop unneeded code examples.

* Formatting

Co-authored-by: Steve Brush <steve.brush@blackbaud.com>

Co-authored-by: Steve Brush <steve.brush@blackbaud.com>

Blackbaud-SteveBrush committed:
chore(release): 5.8.0 (#121)

blackbaud-johnly committed:
docs: remove v4 links from modal docs (#120)

Co-authored-by: John Lyons <John.Lyons@blackbaud.me>

Blackbaud-SteveBrush committed:
chore(release): 5.8.1 (#123)

Blackbaud-SteveBrush committed:
fix(components/core): fix inheritance issue with resize observer media query service (#122) (#124)

Co-authored-by: John White <750350+johnhwhite@users.noreply.github.com>

johnhwhite committed:
fix(components/ag-grid): set lookup cell editor width to match column (#126) (#127)

Blackbaud-SteveBrush committed:
docs: add descriptions for data manager classes (#118) (#134)

Co-authored-by: Paula Hodgkins <77297281+blackbaud-paulah@users.noreply.github.com>

Blackbaud-SteveBrush committed:
fix(components/ag-grid): amend public exports API and JSDocs (#132) (#135)

Blackbaud-SteveBrush committed:
chore(release): 5.8.2 (#136)

Blackbaud-TrevorBurch committed:
docs(apps/code-examples): updates to fix StackBlitz issues (#133) (#137)

Blackbaud-TrevorBurch committed:
refactor(components/data-manager): remove unused title property from the column picker component (#138) (#140)

Blackbaud-SteveBrush committed:
fix(components/popovers): respect alignment and placement values supplied to the popover component (#139) (#141)

If alignment/placement values are not defined by the popover directive, the same values provided to
the component should be respected. Before this change, the values on the component were reset to
their default values.

Blackbaud-SteveBrush committed:
chore(release): 5.8.3 (#142)

blackbaud-johnly committed:
docs(apps/code-examples): update wizard button label (#144)

Co-authored-by: John Lyons <John.Lyons@blackbaud.me>

Blackbaud-SteveBrush committed:
fix(components/theme): add package exports for sky.css (#146)

Blackbaud-SteveBrush committed:
chore(release): 6.0.0-beta.3

Blackbaud-SteveBrush committed:
refactor: adjustments to satisfy TypeScript `noUnusedLocals` rule (#145)

Blackbaud-SteveBrush committed:
docs(components/tiles): fix selectors for drag-and-drop functionality (#143) (#147)

Blackbaud-SteveBrush committed:
chore: `nx migrate 13.10.1` (#149)

Blackbaud-SteveBrush committed:
chore(components/packages): update `@skyux/docs-tools` package group version (#151)

Blackbaud-SteveBrush committed:
fix(components/angular-tree-component): update @circlon/angular-tree-component to 11.0.3 (#150) (#152)

Blackbaud-SteveBrush committed:
chore: add routes for drag-and-drop code examples
